MapboxDirections/MBRouteOptions.swift

```diff
@@ -319,6 +331,14 @@ open class RouteOptions: NSObject, NSSecureCoding {
             params.append(URLQueryItem(name: "roundabout_exits", value: String(includesExitRoundaboutManeuver)))
         }
         
+        if !roadClassesToAvoid.isEmpty {
+            let allRoadClasses = roadClassesToAvoid.description.components(separatedBy: ",")
+            if allRoadClasses.count > 1 {
+                print("`roadClassesToAvoid` only accepts one `RoadClasses`. Although \(roadClassesToAvoid.description) was specified, only \(String(describing: allRoadClasses.first)) will be used.")
```

Comment:
Given the API’s behavior of erroring out on multiple excluded classes, let’s make this an assertion.